.github/workflows/deploy_staging.yml

```diff
@@ -0,0 +1,21 @@
+name: "Build Docker"
+
+on:
+  push:
```

Comment:
Add workflow_dispatch as well

Reply:
added

Reply:
REEUN option is available, It is not required now